### PR TITLE
EffectSsSbn OK

### PR DESCRIPTION
**Pull request by hensldm**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

File: src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.c [hunk 1]
```diff
@@ -1,30 +1,208 @@
 /*
  * File: z_eff_ss_sbn.c
  * Overlay: ovl_Effect_Ss_Sbn
- * Description:
+ * Description: Popped Deku Bubble
+ *
+ * Note: The gfx field is used to store the colPoly, not anything gfx related
+ * This is not a problem since gfx is already void*
```
Comment (EllipticEllipsis) [inline src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.c:7]: Need to remember to review these later.

File: src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.c [hunk 2]
```diff
@@ -1,30 +1,208 @@
...
+    Vec3f bubbleVec;
+    f32 angle;
+    f32 opposite;
+
+    Math_Vec3f_Copy(&this->pos, &initParams->pos);
+    Math_Vec3f_Copy(&this->velocity, &gZeroVec3f);
+    Math_Vec3f_Copy(&this->accel, &gZeroVec3f);
+    this->flags = 0;
+
+    this->gfx = initParams->colPoly;
```
Comment (EllipticEllipsis) [inline src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.c:62]: You may or may not want to note it inline too.

**Review by EllipticEllipsis — APPROVED** [review]

File: undefined_syms.txt [hunk 3]
```diff
@@ -470,10 +470,6 @@ D_0406F380 = 0x0406F380;
 D_0406F9F0 = 0x0406F9F0;
 D_0406FAE0 = 0x0406FAE0;
 D_040706E0 = 0x040706E0;
-D_04071230 = 0x04071230;
-D_04073F00 = 0x04073F00;
-D_04074330 = 0x04074330;
-D_04075400 = 0x04075400;
```
[the remark this answers is not in the document]
Comment (hensldm) [reply undefined_syms.txt:476]: Probably not. TBH, I just like to see `undef_syms` get smaller whenever I get the chance :) .

File: src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.c (hunk 1 above)
Comment (hensldm) [reply src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.c:7]: I do plan on doing a full review/clean up PR once all the effects are in. Still not sure on exact documentation yet. Though I'll add a todo to remind myself to take a look.

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**